server/src/weaverbird/backends/pypika_translator/translators/base.py

```diff
@@ -337,9 +340,11 @@ def get_query_builder(
         query_builder: QueryBuilder | None = None,
         unwrap_last_step: bool = False,
     ) -> QueryBuilderContext:
-        if len(steps) < 0:
-            ValueError("No steps provided")
-        assert steps[0].name == "domain" or steps[0].name == "customsql"
+        if len(steps) <= 0:
+            raise ValueError("No steps provided")
+        assert (
+            steps[0].name == "domain" or steps[0].name == "customsql"
+        ), "First step must be one of domain or customsql step"
```

Comment:
Nice, could you please also add the step name in the assertion message ?

Reply:
[75b5cc8](https://github.com/ToucanToco/weaverbird/pull/2238/commits/75b5cc85f88f163b6c8e32cd51fdaf1f56ff4c56)